client/src/lib.rs

```diff
@@ -49,6 +49,9 @@ use tower::ServiceBuilder;
 use url::Url;
 use uuid::Uuid;
 
+#[cfg(any(feature = "mocks", test))]
```

Comment:
Where are we using this feature?

Reply:
It's needed by the unit tests since the client is a different crate now and needs to export the mock